src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/books/Books';
+import Category from './components/Category/Category';
+
+import Nav from './components/Nav';
+
+const App = () => (
+  <Router>
+    <Nav />
+    <Routes>
+      <Route path="/" element={<Books />} />
+      <Route path="/Category" element={<Category />} />
+    </Routes>
+  </Router>
+
+);
```

Comment:
## Your Design
![image](https://user-images.githubusercontent.com/93039359/232180481-6b2b9163-aad9-4cb2-950e-8b095009d5a9.png)
![image](https://user-images.githubusercontent.com/93039359/232180514-c82e8662-c827-4a23-bf2d-e416c9e769d2.png)

## Zeplin Design
![image](https://user-images.githubusercontent.com/93039359/232180718-335ff610-5e12-46d9-ac26-b326ce204c94.png)

- [ ] It would be nice if all the contents on your webpage should be aligned perfectly because your navbar title is so much on the left side but on the zeplin's design the navbar is very perfectly aligned with the books 👍
- [ ] It would be nice if your form should be perfectly aligned with the books as I already mentioned in the above change to make your design follow Zeplin's design to make your project look attractive and professional 😊